### DISCUSSION TITLE
[Help]: Centering the first card and the last card

### DISCUSSION THREAD

**Discussion by theRolento** [davidjerleke/embla-carousel#935] [Help] [answered]
### Summary

![immagine](https://github.com/davidjerleke/embla-carousel/assets/74258012/5d05cee5-f449-46c0-86c5-5c564621400c)

I have this slider that has 7 cards. The cards are like 600px width fo example.
I need the card1 to start at the center of the row, then when scrolling to the left they **continue** in the left blank space thru the browser margin and end with the card7 at the center of the row again (wih blank space on its right).
I can't figure out how to do it, i think i tried every possible way playing with margins too.
Align option doesn't seem to work (apparently).

As you can see from the picture, the slider starts at the left border with card1 and it ends with card7 at the border right.

```
.embla {
			overflow: hidden;

			.embla__container {
				display: flex;
			}
		}

.slide {
		flex: 0 0 100%;
		min-width: 0;
	}
```

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [X] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by sarussss** [comment, marked as answer]
Hi @theRolento 
You can try option [containscroll](https://www.embla-carousel.com/api/options/#containscroll)
`containscroll: false`
[Example](https://codesandbox.io/s/nzdm3r?file=/src/js/index.js)

**Reply by theRolento** [reply]
> Hi @theRolento You can try option [containscroll](https://www.embla-carousel.com/api/options/#containscroll) `containscroll: false` [Example](https://codesandbox.io/s/nzdm3r?file=/src/js/index.js)

OH MY GOD i can't believe it was so much simple...
The complicated way i found was to make two dummy empty divs around the cards and making a function for "auto-width them"...

Thank you!
